File: src/ai/Evaluation2.java
package ai;

import game.Bot;

public class Evaluation2 implements Evaluation {
    @Override
    public int evaluate(Bot bot, int nbSeedCapturedByMe, int nbSeedCapturedByOpponent, int[][] board) {

        int diffNbSeedCaptured = nbSeedCapturedByMe - nbSeedCapturedByOpponent;
        int diffMobility = bot.mobility(board) - bot.getOpponent().mobility(board);

        // others variables (to complete)

        return diffNbSeedCaptured * 3 + diffMobility;
    }
}

File: projectC/awale/src/ai/evaluation2.c
#include "../../headers/ai/evaluation2.h"

int evaluate_evaluation2(Player player, int nbSeedCapturedByMe, int nbSeedCapturedByOpponent, int board[16][3])
{
    int diffNbSeedCaptured = nbSeedCapturedByMe - nbSeedCapturedByOpponent;
    int diffMobility = mobility(&(player), board) - mobility(player.opponent, board);

    // Others variables (to complete)

    return diffNbSeedCaptured * 3 + diffMobility;
}
